xla/service/gpu/build_defs.bzl

```diff
@@ -1,8 +1,8 @@
 """ GPU-specific build macros.
 """
 
-load("@local_config_cuda//cuda:build_defs.bzl", "cuda_library")
-load("@local_config_rocm//rocm:build_defs.bzl", "rocm_copts")  # copybara:comment
+load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda_is_configured", "cuda_library")
+load("@local_config_rocm//rocm:build_defs.bzl", "if_rocm_is_configured", "rocm_copts")  
```

Comment:
I think we need the `# copybara:comment` back on line 5, but I think that should be it. I think any problem after this on import should be for me to figure out